dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -294,4 +294,60 @@
             CREATE UNIQUE INDEX case_insensitive_workflow_workflowname on workflow using btree (sourcecontrol, organization, repository, lower(workflowname)) where workflowname is not null and id > 18876;
         </sql>
     </changeSet>
+    <changeSet author="ktran (generated)" id="modifyAndAddColumnsForMetrics">
+        <addNotNullConstraint columnDataType="varchar(255)" columnName="average" tableName="cpu_metric"/>
```

Comment:
> Changed the data type for a few of the Metrics classes because I realized it made more sense while working on the metrics aggregation in dockstore-support  

In that case, shouldn't these be float/double as well ?

Reply:
Good catch, I was updating the migrations file incrementally so the column data type hadn't changed yet at that point